Unify Formatting (#82)
C2Devel/terraform-provider-croccloud: website/docs/d/network_interface.md

layout: "aws"
page_title: "aws_network_interface"
description: |-
  Provides information about a network interface.
---

[describe-network-interfaces]: https://docs.k2.cloud/en/api/ec2/network_interfaces/DescribeNetworkInterfaces.html

> [tf-network-interface]: network_interface.html

Reviewer: Не используется, можно удалить

The following arguments are supported:

> * `id` – (Optional) The identifier for the network interface.

Reviewer: Аргументы не отсортированы

* `filter` – (Optional) One or more name/value pairs to use as filters.

>     * _Valid values:_ See supported names and values in [EC2 API documentation][describe-network-interfaces]

Reviewer: Нужно перенести определение ссылки в начало страницы

The following arguments are supported:

* `filter` - (Optional) One or more name/value pairs to use as filters.
    * _Valid values:_ See supported names and values in [EC2 API documentation][describe-network-interfaces]

> * `id` - (Optional) The identifier for the network interface.

Reviewer: ```suggestion
* `id` - (Optional) The ID of the network interface.
```

## Attribute Reference

### Supported attributes

See the [`aws_network_interface`](network_interface.md) for details on the returned attributes.

Additionally, the following attributes are exported:

* `arn` - The Amazon Resource Name (ARN) of the network interface.

> * `association` - The association information for an Elastic IP address (IPv4) associated with the network interface. See supported fields below.

Reviewer: ```suggestion
* `association` - The association information for an Elastic IP address (IPv4) associated with the network interface.
  The structure of this block is [described below](#association).
```

* `private_ips` - The private IPv4 addresses associated with the network interface.
* `security_groups` - The list of security groups for the network interface.
* `subnet_id` - The ID of the subnet.
* `tags` - Map of tags assigned to the network interface.
* `vpc_id` - The ID of the VPC.

> #### `association`

Reviewer: ```suggestion
#### association
```

### Unsupported attributes

~> **Note** These attributes may be present in the `terraform.tfstate` file, but they have preset values and cannot be specified in configuration files.

The following attributes are not currently supported:

> `interface_type`, `ipv6_addresses`, `requester_id`, `outpost_arn`, `association.carrier_ip`.

Reviewer: `association.carrier_ip` не отсортировано

> ## Import

Reviewer: Блок импорта можно полностью удалить, он тут по ошибке
